pygmt/helpers/decorators.py

```diff
@@ -63,8 +63,8 @@
             :gmt-docs:`gmt.html#xy-full`.
          """,
     "a": r"""
-        aspatial : str
-            [*col*\ =]\ *name*\ [,...].
+        aspatial : bool or str
+            [[*col*\ =]\ *name*\ [,...]].
```

Comment:
I think we should revert the changes. `[ ]` means the whole argument is optional, so it's equivalent to `aspatial=True` or `aspatial=""`. 
```suggestion
            [*col*\ =]\ *name*\ [,...].
```

Reply:
I do not understand. The outer brackets are what indicate that `aspatial=""` is allowed, so I do not see why they should be removed. It does not seem to me that `bool` alone is sufficient, because I would think that this
```
aspatial : bool or str
    [*col*\ =]\ *name*\ [,...].
```
would mean that ``aspatial=True`` is allowed, but if a str is passed it must contain arguments (i.e., `aspatial=""` is not allowed).

Reply:
Both `aspatial=""` and `aspatial=True` are converted to GMT arguments `-a` internally, so they're the same. Similarly, `frame=""` is also allowed, but we use `frame=True` in all the examples, perhaps `frame=True` is more readable?

Reply:
OK, I will revert the change to be consistent with the other parts of the PyGMT documentation, even though it still seems to me that including the brackets is more technically correct because while `parameter=""` and `parameter=True` internally are the same, booleans and strings are not the same to the external user.